src/strkey.rs

```diff
@@ -17,103 +17,112 @@ pub enum Strkey {
     PrivateKeyEd25519(StrkeyPrivateKeyEd25519),
 }
 
-impl Strkey {
-    pub fn to_string(&self) -> String {
```

Comment:
Could we keep the inherent implementations and have the trait implementations call the inherents?

The main reason we had non-trait implementations is I was hoping to make the inherent implementations `const` so that we could use the functions in const contexts. I'd like to retain the possibility of doing that. As a bonus, keeping the inherents would make this change additive rather than breaking.